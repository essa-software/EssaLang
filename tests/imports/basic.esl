import a;

func main() {
    hello(1234);
}
