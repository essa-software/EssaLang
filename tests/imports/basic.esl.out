Hello, 1234, from Test!